modules/gitops-configuring-high-availability-for-argo-rollouts.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/enabling-ha-support-for-argo-rollouts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-configuring-high-availability-for-argo-rollouts_{context}"]
+= Configuring high availability for Argo Rollouts
+
+To enable high availability, configure the `ha` specificaiton in the `RolloutManager` CR by completing the following steps:
+
+.Procedure
+
+.  Log in to the {OCP} web console as a cluster administrator. 
+
+. In the *Administrator* perspective, click *Operators* -> *Installed Operators*.
+
+. Create or select the project where you want to create and configure a `RolloutManager` custom resource (CR) from the *Project* drop-down menu.
+
+. Select *{gitops-title}* from the installed operators.
+
+. In the *Details* tab, under the *Provided APIs* section, click *Create instance* in the *RolloutManager* pane.
+
+. On the *Create RolloutManager* page, select the *YAML view* and edit the YAML.
```

Comment:
Nit: is `select the *YAML view*` correct? Should it be `select *YAML view*` or `select the *YAML* view`? (Might be worth double-checking.)

Reply:
It is select the YAML view as per step 6 of this [procedure](https://docs.openshift.com/gitops/1.14/argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.html#gitops-creating-rolloutmanager-custom-resource_using-argo-rollouts-for-progressive-deployment-delivery). I have reused content from this procedure because we need to follow steps from this section.